pkg/workloadmeta/collectors/containerd/containerd.go

```diff
@@ -193,7 +194,7 @@ func (c *collector) generateInitialEvents(ctx context.Context) ([]containerdeven
 
 		ignore, err := c.ignoreEvent(ctx, &event)
 		if err != nil {
-			return nil, err
+			log.Debugf("error while deciding to ignore event, keeping it: %s", err)
```

Comment:
I think it'd be good to add a comment here like the one you have in the PR description so we remember why returning an error here becomes a problem.

I agree that it's better to keep the event if we don't know if it should be ignored. Although I'm not totally sure if keeping the pause containers here for example might cause some side-effects.